node/pkg/governor/governor_monitoring.go

```diff
@@ -431,4 +449,152 @@ func (gov *ChainGovernor) CollectMetrics(hb *gossipv1.Heartbeat) {
 	}
 
 	metricTotalEnqueuedVAAs.Set(float64(totalPending))
+
+	if startTime.After(gov.nextConfigPublishTime) {
+		gov.publishConfig(hb, sendC, gk, ourAddr)
+		gov.nextConfigPublishTime = startTime.Add(time.Minute * time.Duration(5))
+	}
+
+	if startTime.After(gov.nextStatusPublishTime) {
+		gov.publishStatus(hb, sendC, startTime, gk, ourAddr)
+		gov.nextStatusPublishTime = startTime.Add(time.Minute)
+	}
+}
+
+var governorMessagePrefix = []byte("governor|")
+
+func (gov *ChainGovernor) publishConfig(hb *gossipv1.Heartbeat, sendC chan []byte, gk *ecdsa.PrivateKey, ourAddr ethCommon.Address) {
+	chains := make([]*gossipv1.ChainGovernorConfig_Chain, 0)
+	for _, ce := range gov.chains {
+		chains = append(chains, &gossipv1.ChainGovernorConfig_Chain{
+			ChainId:            uint32(ce.emitterChainId),
+			NotionalLimit:      ce.dailyLimit,
+			BigTransactionSize: ce.bigTransactionSize,
+		})
+	}
+
+	tokens := make([]*gossipv1.ChainGovernorConfig_Token, 0)
+	for tk, te := range gov.tokens {
+		price, _ := te.price.Float32()
+		tokens = append(tokens, &gossipv1.ChainGovernorConfig_Token{
+			OriginChainId: uint32(tk.chain),
+			OriginAddress: "0x" + tk.addr.String(),
+			Price:         price,
+		})
+	}
+
+	gov.configPublishCounter += 1
```

Comment:
Will this increment unbounded?

Reply:
I guess after 2^64 five minute intervals it will wrap around. . .